src/js/components/List/List.js

```diff
@@ -357,13 +366,19 @@ const List = React.forwardRef(
                       // accessibility guidelines that focus remains on `ul`
                       listRef.current.focus();
                     },
-                    onMouseOver: () => setActive(index),
-                    onMouseOut: () => setActive(undefined),
+                    onMouseOver: () => {
+                      setLastActive(undefined);
+                      setActive(index);
+                    },
+                    onMouseOut: () => {
+                      setActive(lastActive);
```

Comment:
This doesn't seem right. I would think we would leave `lastActive` alone, to be used in the next `onFocus`, but we wouldn't want to `setActive` when the mouse goes away.

Reply:
As soon as the modal / Layer opens, `onMouseOut` is triggered and upon closing, I chose to `setActive`  to what `lastActive` had stored as the way to keep an item active when a modal / Layer closes.